src/test/itemsCounter.test.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const fetchMock = require('fetch-mock');
+
```

Comment:
> OPTIONAL

Kindly, for next project think to use beforeEach() or afterEach() hooks to ensure that fetchMock is properly reset before or after each test, respectively. This can help prevent any test dependencies or conflicts.

Like this 

> describe('async fetch', () => {
>   beforeEach(() => {
>     fetchMock.reset();
>   });
> 
>   test('can fetch', async () => {
>    .
      .
      .
>   });
> 
>   afterEach(() => {
>     fetchMock.reset();
>   });
> });